internal/database/database.go

```diff
@@ -22,7 +22,11 @@ func SetupDb(cfg *configs.Config) {
 		panic("Error opening the database.")
 	}
 
+	_ = db.AutoMigrate(&model.Booking{})
 	_ = db.AutoMigrate(&model.User{})
+	_ = db.AutoMigrate(&model.Chat{})
+	_ = db.AutoMigrate(&model.RequestQuery{})
+	_ = db.AutoMigrate(&model.Message{})
```

Comment:
Can consider using https://github.com/rubenv/sql-migrate for more flexible migrations. GORM wont [delete columns](https://gorm.io/docs/migration.html#Auto-Migration)

Reply:
interesting! added migration files and ~~yoinked your create domain code~~

Reply:
:100: nice